rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -237,11 +237,10 @@ SecRule REQUEST_METHOD "@rx ^POST$" \
     ver:'OWASP_CRS/3.2.0',\
     severity:'WARNING',\
     chain"
-    SecRule &REQUEST_HEADERS:Content-Length "@eq 0" \
-        "chain"
-        SecRule &REQUEST_HEADERS:Transfer-Encoding "@eq 0" \
-            "setvar:'tx.anomaly_score_pl1=+%{tx.notice_anomaly_score}'"
-
+    SecRule REQUEST_METHOD "@streq POST" "chain"
```

Comment:
I'm OK with doing this change now but I'd prefer if changes unrelated to what this PR is fixed are done (and documented) separately in case people wants to understand why.

Reply:
Yes, you're right - but I think that the replacing of `@rx ^POST$` by `@streq POST` seems "just" a _code cosmetics_ :).

Let's see other opinions.